civitai-downloader-v2/tests/api/test_downloads.py

```
@@ -0,0 +1,230 @@
+"""
+Test download management API endpoints.
+"""
+import pytest
```

Ruff (0.12.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)